README.md

```diff
@@ -197,3 +216,4 @@ Additional Ethereum Implementations
 - [Go-Ethereum (Geth)](https://github.com/ethereum/go-ethereum/)
 - [EthereumJS](https://github.com/ethereumjs/ethereumjs-vm)
```

Comment:
Maybe we should remove other implementations? Because we should be promoting out client

Reply:
Ok, I'm fine with removing.